lib/overloads.js

```diff
@@ -2,20 +2,20 @@
 
 const { areDistinguishable, sameType } = require("./types");
 
+/**
+ * @param {string} type
+ * @param {*} A
+ * @param {import("./constructs/interface")} I
+ * @return {import("webidl2").OperationMemberType[] | import("webidl2").ConstructorMemberType[]}
+ */
```

Comment:
Please don't add this to your PRs; it makes them hard to review.

Reply:
In&nbsp;what&nbsp;way does&nbsp;**JSDoc** make&nbsp;code harder&nbsp;to&nbsp;review?

Reply:
It's a bunch of meaningless noise for humans. I understand it makes some software happier but this is unreadable.

Reply:
It’s&nbsp;meaningful to&nbsp;humans&nbsp;too, [as&nbsp;it&nbsp;describes what&nbsp;the&nbsp;function&nbsp;does](https://jsdoc.app/about-getting-started.html).